requirements.txt

```diff
@@ -25,4 +25,4 @@ google-api-core
 boto3==1.24.7
 fsspec==2022.5.0
 s3fs==2022.5.0
-pandas
+pandas
```

Comment:
not sure what github thinks the difference is between `pandas` and `pandas`... 
would be useful to add `https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.1.0/en_core_web_sm-3.1.0.tar.gz` to the reqs, I didn't have the model downloaded

Reply:
truly riddle me that